elevate-cpanel

```diff
@@ -230,15 +230,13 @@ sub _OPTIONS {
     return qw( service start clean continue manual-reboots status log check skip-cpanel-version-check skip-elevate-version-check );
 }
 
-my $logger;
-
 exit( __PACKAGE__->new(@ARGV)->run() // 0 ) if !caller;
 
 sub run ($self) {
 
     local $| = 1;
 
-    $logger = _init_logger();
+    _init_logger();
```

Comment:
does this release the logger?

Reply:
Maybe change the return of _init_logger to be empty if you think it has no value?

Reply:
Technically, `Log::Log4perl->init()` has a return value hard-coded to `1` via `Log::Log4perl::Config::_init`. I figure that this can be addressed if/when we change everything over in #83.